pipeline/.env.example

```
@@ -41,3 +41,6 @@ PROJECT_DB_PASS={{PROJECT_DB_PASS}}
 PROJECT_LOG_LEVEL=warning
 PROJECT_PORT=8084
 PROJECT_ITEMS_LIMIT_FOR_LIST=1000
+
+JSON_SCHEMAS_PATH='/home/user/svetit/doc/api/schemas/'
```

Нужно использовать относительный путь
```suggestion
JSON_SCHEMAS_PATH='../../../../schemas/'
```